spring-cloud-gcp-docs/src/main/asciidoc/core.adoc

```diff
@@ -82,7 +82,7 @@ spring.cloud.gcp.credentials.location=file:/usr/local/key.json
 ----
 
 If that property isn't specified, the starter tries to discover credentials from a
-https://github.com/google/google-auth-library-java/blob/master/oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java#L57[number of places]:
+https://github.com/google/google-auth-library-java/blob/a8929df490d3751f2349986264290a074dee35e1/oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java#L57[number of places]:
```

Comment:
Not sure what you mean here.

Code lines are necessary because it's a big file and we want the reader to jump to the important lines.
But the line location can change in between commits, which is why pinning to one commit is useful.

Reply:
Well, that's exactly I would prefer to stay away of linking to the target code. When that code is changed, that commit and, therefore, this link becomes obsolete and may cause a confuse for end-users to track the proper code. 
And again: when that code is changed, you will have to change this doc as well. Would be better to keep the code as stable as possible and concentrate on the new features and don't spend time on always polishing like this. 

We really do here too much changes without value for target users of our project...

Reply:
+1 for not linking to the code. The same information is contained in docs.
Here's one place: https://developers.google.com/identity/protocols/application-default-credentials
Here's another: https://github.com/GoogleCloudPlatform/google-cloud-java#authentication

Reply:
That's great! That's exactly what I was looking for.
I may address it on merge though since Stephane is on vacation until the next year.

Just let me know! Or feel free do that yourself

Reply:
OK great! If you could do that, it'd be great @artembilan. Thanks!

Reply:
Agreed that not linking to the code at all is better.